persistence/persistence-mybatis/src/main/resources/org/cbioportal/persistence/mybatis/MolecularProfileMapper.xml

```diff
@@ -145,5 +145,4 @@
                          genetic_profile.STABLE_ID = #{referredMolecularProfileId}
                        )
     </select>
-    
```

Comment:
Only whitespace changes in this file. Remove from PR.

Reply:
Done!